### ISSUE TITLE
About Training 

### ISSUE THREAD

**Issue by ldfinfontainebleau** [wdjang/BRS-Interactive_segmentation#2]
HI, I have used your great work on our own datasets(cloth segmentation), unfortunately, it happens pretty much bad cases. I want to figure it out, was it about the training data. Would you please update your training files if it is possible. Thank you.

**Comment by wdjang** [comment]
Hi, could you let me know more details about the training problem?

**Comment by ldfinfontainebleau** [comment]
> Hi, could you let me know more details about the training problem?

Of course! According to your paper, I speculate a bad case is more than 20 clicks or can not shape the object properly. Here I will sent some examples below: 
![52519122-5701-48d9-82F4-296C0C6CCE1A](https://user-images.githubusercontent.com/17682537/57340713-99f53700-7169-11e9-9e6f-53d9b2383ed2.png)
![89692AC6-6FE8-4038-8345-D0F342AD6424](https://user-images.githubusercontent.com/17682537/57340714-9a8dcd80-7169-11e9-9214-fbaff232f04b.png)

**Comment by wdjang** [comment]
Thanks for the examples. The training dataset doesn't have objects like them. You may need to finetune the network by constructing your own dataset. I can upload the training script, but I am not sure when it would be. I need some time to clean up the scripts.

**Comment by ldfinfontainebleau** [comment]
Thank you very much!!! I tried to write train-val.prototxt this days and I have a question. The iact_map seems have two channels contain foreground and background points. Would you please show me what's the data format? Thank you.

**Comment by wdjang** [comment]
It is float32. We compute it as `net_iactmap' in BRS_main.py file.
